pkg/api/blocks/v1.go

```diff
@@ -84,6 +88,8 @@ func (bapi *BlocksAPI) Register(r *route.Router, tracer opentracing.Tracer, logg
 
 	r.Get("/blocks", instr("blocks", bapi.blocks))
 	r.Post("/blocks/mark", instr("blocks_mark", bapi.markBlock))
+	r.Get("/flags/enableMarkDeletion", instr("flags_enableMarkDeletion", bapi.enableMarkDeletionFlagHandler))
+	r.Get("/flags/enableMarkNoCompaction", instr("flags_enableMarkNoCompaction", bapi.enableMarkNoCompactionFlagHandler))
```

Comment:
Can we not add additional API for this? Maybe frontend can just fetch `/flags` and get the value it needs?